### PR TITLE
APPS-1858 TreeTurnaroundTime threshold

### PR DESCRIPTION
**Pull request by Gvaihir**
Changes needed for dxCompiler

### CODE REVIEW

File: api/src/main/scala/dx/api/DxApplet.scala [hunk 1]
```diff
@@ -22,24 +22,26 @@ case class DxAppletDescribe(project: String,
                             runSpec: Option[JsValue] = None,
                             access: Option[JsValue] = None,
                             ignoreReuse: Option[Boolean] = None,
-                            hidden: Option[Boolean] = None)
+                            hidden: Option[Boolean] = None,
+                            treeTurnaroundTimeThreshold: Option[Long] = None)
```
Comment (commandlinegirl) [inline api/src/main/scala/dx/api/DxApplet.scala:30]: Can you add a docstring or comment on what this input is?

**Review by commandlinegirl — DISMISSED** [review]
LGTM modulo a minor request

File: api/src/main/scala/dx/api/DxApplet.scala (hunk 1 above)
Comment (commandlinegirl) [reply api/src/main/scala/dx/api/DxApplet.scala:30]: Here and in other places where needed.

Comment (Gvaihir) [reply api/src/main/scala/dx/api/DxApplet.scala:30]: Added in [2072d6d](https://github.com/dnanexus/dxScala/pull/123/commits/2072d6debfbbfbc12d29e76767905406f4145daf)

**Review by commandlinegirl — APPROVED** [review]

**Pull request merged**